src/ua_model.cpp

```diff
@@ -145,6 +140,9 @@ ua_node_id parse_node_id(const std::string &s)
         return ua_node_id(namespace_id, std::get<int>(address));
     else if (std::holds_alternative<std::string>(address))
         return ua_node_id(namespace_id, std::get<std::string>(address));
+    else
+        // Do somethings correct:
+        return ua_node_id();
```

Comment:
Probably even more correct would be to throw something here. Anything we are trying to create is wrong anyways and an exception might show that best for the moment.
```suggestion
        throw std::invalid_argument(std::string("String contains an invalid NodeId: ") += s);
```